_pages/sponsors.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Sponsors
+layout: single
+excerpt: "EACL 2023 Sponsors."
+permalink: /sponsors
+sidebar:
+  nav: faq
+toc: true
+toc_only: true
+toc_sticky: true
+toc_icon: "cog"
+diamond:
+  - url: https://www.grammarly.com/jobs?utm_source=website&utm_medium=conference_rm&utm_campaign=eacl2023_website 
+    image_path: /assets/images/logos/grammarly.png
+    alt: Grammarly
+    title: Grammarly
+  - url: https://www.liveperson.com/ 
+    image_path: /assets/images/logos/liveperson.jpg
+    alt: LivePerson
+    title: LivePerson
+platinum:
+  - url: https://www.amazon.science/
+    image_path: /assets/images/logos/amazon.png
+    alt: Amazon
+    title: Amazon
+  - url: https://www.bloomberg.com/company/values/tech-at-bloomberg/artificial-intelligence-ai/
+    image_path: /assets/images/logos/bloomberg.png
+    alt: Bloomberg
+    title: Bloomberg
+silver:
+  - url: https://jobs.duolingo.com/ 
+    image_path: /assets/images/logos/duolingo.png
+    alt: Duolingo
+    title: Duolingo
+bronze:
+  - url: https://research.adobe.com/
+    image_path: /assets/images/logos/adobe.png
+    alt: Adobe
+    title: Adobe
+  - url: https://research.adobe.com/
```

Comment:
Does this link need to be the one for babelscape?

Reply:
Ah, nice catch! Fixed :)